reservation_units/tests/test_admin.py

```diff
@@ -30,7 +31,7 @@ def test_reservation_kind_is_direct_are_excluded_from_queryset(self):
             },
         )
 
-        queryset, _ = self.radmin.get_search_results(get_request, self.queryset, "")
```

Comment:
Why the `self` reference does not work anymore?

Reply:
I couldn't fully figure that out. The error message I got was:
```
cannot pickle 'module' object
```

Based on my searches it seems to be somehow related to Python not being able to serialize the class or something like that. It was kinda weird and I couldn't figure out any proper way to fix it. I tried to debug the test and the class instance was initialized properly in the setup. But as soon as we hit the test and tried to access the instance, we got the error. We didn't have any issues with the other variables.

If you can figure out a proper fix for this, please let me know 😄

Reply:
Okay, I also did some quick tryouts for it and couldn't figure out main reason but noticed that the field works if put in `setUp` which is instance method.

Reply:
This was an awesome find! I changed the code so that the object is initialized in `setUp`.